src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const styleArray = sourceString
+    .split(';')
+    .map(style => style.trim())
+    .filter(style => style.length > 0)
+    .map(style => style.split(':'));
+
+  for (const [property, value] of styleArray) {
+    styleObject[property.trim()] = value.trim();
+  }
```

Comment:
You can combine all of these lines either in one `forEach` or `reduce`.
1. Start the `forEach`
2. Split by `:`
3. Check if both property and value are defined
4. Add the style to `styleObject`. (Don't forget to trim everything)